backend/config/routes.rb

```diff
@@ -16,6 +16,9 @@
       end
       post '/refresh', to: 'sessions#refresh'
       post 'hello', to: 'sessions#hello'
+      resources :groups do
```

Comment:
This is not correct. Feel free to check [this article](https://guides.rubyonrails.org/routing.html) and fix it